domain/controllernode/service/service.go

```diff
@@ -17,6 +18,7 @@ type State interface {
 	CurateNodes(context.Context, []string, []string) error
 	UpdateDqliteNode(context.Context, string, uint64, string) error
 	SelectDatabaseNamespace(context.Context, string) (string, error)
+	SetRunningAgentBinaryVersion(ctx context.Context, controllerID string, version coreagentbinary.Version) error
```

Comment:
nit: the other methods don't use named params so good to be consistent

Reply:
Sure thanks ian :)